MapboxNavigationTests/Sources/Tests/NavigationViewControllerTests.swift

```diff
@@ -87,13 +89,18 @@ class NavigationViewControllerTests: XCTestCase {
         XCTAssertFalse(wayNameView.isHidden, "WayNameView should be visible.")
     }
     
+    // TODO: This test doesn't seem to be testing anything. Adding a test where `updatedStyleNumberOfTimes = 1` might be illuminating
     func testNavigationShouldNotCallStyleManagerDidRefreshAppearanceMoreThanOnceWithOneStyle() {
```

Comment:
These tests seem a little... weird. I left some comments expressing my confusion, but opted not to try to change them more than necessary in this PR. I don't think I made them any more weird.

Reply:
Thanks, yeah, some of the tests are indeed weird. We should be fine to improve them later on 👍

Reply:
I am not sure if we should add all the TODOs in the code though? Maybe you could summarise your findings in an issue and we can look at what happens exactly?